### DISCUSSION TITLE
Include Hudsucker Licenses

### DISCUSSION THREAD

**Discussion by roniemartinez** [emanuele-em/proxelar#62] [General]
Just noticed that a few of Hudsucker source code is being used in this library without including the copyrights or linking to the repo of the code owner (Apache and MIT requires it).

Example:

https://github.com/emanuele-em/man-in-the-middle-proxy/blob/90ae39958d5ec223359ccd3c8a15cced4b93a1c4/proxyapi/src/proxy/internal.rs

https://github.com/omjadas/hudsucker/blob/0d32afbb66780c09b3369f9dea904df828b85d94/src/proxy/internal.rs

**Comment by emanuele-em** [comment]
Hi! @roniemartinez thank you very much! I added all attributions!
